gym_adserver/wrappers/compare_agents.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+
+import gym
+from gym import logger, spaces
+from joblib import Parallel, delayed
+
+from gym_adserver.agents.epsilon_greedy_agent import EpsilonGreedyAgent
+from gym_adserver.agents.random_agent import RandomAgent
+from gym_adserver.agents.softmax_agent import SoftmaxAgent
+from gym_adserver.agents.ucb1_agent import UCB1Agent
+
+def setup_environment(env_name, num_ads, time_series_frequency):
+    env = gym.make(env_name, num_ads=num_ads, time_series_frequency=time_series_frequency)
+    env.seed(args.seed)
+    return env
+
+def render_environment(env, i, **kwargs):
+    if kwargs['output_file'] is not None:
+        kwargs['output_file'] = str(i) + "_" + kwargs['output_file']
+    env.render(**kwargs)
+    env.close()
```

Comment:
@roelbertens do you see any drawback in this refactoring?

Reply:
Nice addition!